i18n/states/PY.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Paraguay states
+ *
+ * @package WooCommerce/i18n
+ * @version 2.0.0
+ */
+
+global $states;
+
+defined( 'ABSPATH' ) || exit;
+
+$states['PYG'] = array(
+    'PY-16' => __( 'Alto Paraguay', 'woocommerce' ),
+    'PY-10' => __( 'Alto Paran&aacute;', 'woocommerce' ),
+    'PY-13' => __( 'Amambay', 'woocommerce' ),
+    'PY-ASU' => __( 'Asunci&oacute;n', 'woocommerce' ),
+    'PY-19' => __( ' Boquer&oacute;n', 'woocommerce' ),
```

Comment:
Curious where you got the state codes from? According to Wikipedia this should be PY-17, there are a couple other codes that differ here from the Wikipedia page for Paraguay https://en.wikipedia.org/wiki/Paraguay#Administrative_subdivisions

Reply:
I get from https://en.wikipedia.org/wiki/ISO_3166-2:PY probably is incorrect because not have PY-18, I not live there, I'm brazilian --'

working...